docs/website/static/community.html

```diff
@@ -337,6 +347,72 @@ <h2 class="divider line glow">Contribute</h2>
 
           </div>
 
+          <h2 class="divider line glow">Meetups</h2>
+
+          <div class="grid-container">
+
+            <div class="card">
+              <div class="card-body">
+                <div class="support-card-content d-flex flex-column align-items-start">
+                  <img title="Amsterdam OPA Users" class="card-img align-middle" src="img/community-logos/meetup.png"/>
+                  <div class="card-text">
+                    <div class="header">Amsterdam OPA Users</div>
+                    <br/>
+                    <a href="https://www.meetup.com/opa-amsterdam/"
+                       class="btn btn-primary btn-lg" role="button">Join Amsterdam OPA users</a>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="card-body">
+                <div class="support-card-content d-flex flex-column align-items-start">
+                  <img title="London OPA Meetup" class="card-img align-middle" src="img/community-logos/meetup.png"/>
+                  <div class="card-text">
+                    <div class="header">London OPA Meetup</div>
+                    <br/>
+                    <a href="https://www.meetup.com/london-opa-meetup/"
+                       class="btn btn-primary btn-lg" role="button">Join London OPA Meetup</a>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="card-body">
+                <div class="support-card-content d-flex flex-column align-items-start">
+                  <img title="Stockholm OPA Users" class="card-img align-middle" src="img/community-logos/meetup.png"/>
+                  <div class="card-text">
+                    <div class="header">Stockholm OPA Users</div>
+                    <br/>
+                    <br/>
+                    <div class="h-50"></div>
+                    <a href="https://www.meetup.com/stockholm-opa-meetup/"
+                       class="btn btn-primary btn-lg" role="button">Join Stockholm OPA Users</a>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="card">
+              <div class="card-body">
+                <div class="support-card-content d-flex flex-column align-items-start">
+                  <img title="Meetup" class="card-img align-middle" src="img/community-logos/meetup.png"/>
+                  <div class="card-text">
+                    <div class="header">Host your own!</div>
+                    <div class="subheader">
+                      Host a localized meetup in your area or virtually. Share what you’ve learned and connect with the OPA community!
```

Comment:
[nit] That's a funky single-quote sign there. I guess it's not wrong, just have a tendency to use the simple one, `'`

Reply:
That section was merely moved in this PR, but yeah, I'll have that fixed while I'm at it 👍

Reply:
Thanks!

Reply:
Fixed 👍